contracts/mocks/MockVotesOracle.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/utils/Context.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+// MockVotesOracle contract.
+contract MockVotesOracle is Context, Ownable {
```

Comment:
question: Why use the prefix `Mock` instead of `Stub`?

As the behaviour of the contract is fixed (rather than dynamic), wouldn't it be a stub?
e.g. https://enterprisecraftsmanship.com/posts/stubs-vs-mocks/

Reply:
I can replace with Stub, no problem or just make it concrete VotesManager or VotesOracle.

Reply:
A simple concrete contract sounds good ...either of those names work 👍